native/jest.config.js

```diff
@@ -10,6 +10,7 @@ const transformNodeModules = [
   'hashids',
   '@react-native-community/progress-bar-android',
   '@react-native-community/progress-view',
+  '@react-native-community/picker',
```

Comment:
Do we use `@react-native-community/picker` or `@react-native-picker/picker` now? There are usages of both

Reply:
The latter one. They recently moved to another repo. Must have missed the occurence of `@react-native-community/picker`